### ISSUE TITLE
Wishart and MvNormal

### ISSUE THREAD

**Issue by piantado** [pymc-devs/pymc3#108]
I am having problems using Wishart as a parent of MvNormal. Basically, no proposals are accepted (although if I burn in for longer, about 20k, it will tune the step size down to about 10^13 and accept). Code is below. If you comment out the observed data,  d=pymc.MvNormal... , it seems to do MCMC on the wishart prior okay. 

```
import numpy
import pymc

N = 2

def make_model():

    wish = pymc.Wishart('wish', n=N, Tau=numpy.eye(N) )

    d = pymc.MvNormal('d', numpy.zeros(N), wish, value=numpy.array([1.0, 1.0]), observed=True)
    return locals()

M = pymc.Model(make_model())
S = pymc.MCMC(M)
S.sample(iter=2000, burn=1000, thin=1, verbose=True)
print S.trace('wish')[0]
print S.trace('wish')[:] - S.trace('wish')[0]
```

**Comment by fonnesbeck** [comment]
This is because your degrees of freedom parameter needs to be at least k+1, where k is the dimension of the multivariate normal. It runs fine for me when you specify n=3, for example.

**Comment by fonnesbeck** [comment]
Sorry, I am getting a failure here as well. I will take a closer look.

**Comment by fonnesbeck** [comment]
This should be fixed in 1623147ff1eb70dda701e65c5d5e811634abfb72

**Comment by piantado** [comment]
Awesome, works! Thank you!
